Form-Controls/index.html

```diff
@@ -9,20 +9,74 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
+ 
+  
+
+   <div class="wrapper">
+
+      <header>
+        <h1>Product Pick</h1>
+      </header>
+      
+      <p>Please fill out the form below to place your order.</p>
+      <form>
+        <fieldset>
+          <legend>Personal details</legend>
+          <label for="name"> Your name</label>
+          <input type="text" id="name" name="name">
+          <label for="phone">Phone number</label>
+          <input type="tel" id="phone" name="phone">
+          <label for="email">email</label>
+          <input type="email" id="email" name="email">
```

Comment:
I think there are some validations missing on this form. What do you think? What attributes could you add to make this form work better?

Reply:
Hi @SallyMcGrath Thank you for your feedback. I added changes on date by using max and min attribute. Can you please approve if I have done it correctly? Thank you.